linopy/solvers.py

```diff
@@ -109,8 +109,6 @@
     import coptpy
 
     with contextlib.suppress(coptpy.CoptError):
```

Comment:
```suggestion
    available_solvers.append("copt")
```

Please remove the check then as well